### ISSUE TITLE
Possibility to limit minimum extrusion lenght

### ISSUE THREAD

**Issue by burner-** [alexrj/Slic3r#4422]
With big (example 1mm) nozzle & bowden you can end up to get big blobs or even failed prints because slic3r adds too small details to infills, slopes and solid surfaces even object it self does not include small details. 

I suggest to allow user make two limitations:
- Minimum non visible extrusion
  * this will allow gaps in solid infills etc because in infill it is better to get gap than try fill too small area and get blobs where extruder hits at next round. Currently looks that there is no way to prevent this problem. 

- Minimum visible extrusion
 *this will fix problem where certain kind slopes can cause hanging blobs etc problems. 

Maybe if it is easy to make then split that visible extrusion to:
- Minimum extrusion at perimeter
- Minimum extrusion at slope wall
- Minimum extrusion at top layer

**Comment by lordofhyphens** [comment]
Turn off gap fill.

**Comment by burner-** [comment]
Yes it is already turned off and it does not fix issue. Naturally it helps little because its one part of problem but not fully.

**Comment by lordofhyphens** [comment]
Please add a screenshot of Slic3r's 3d preview, a photo of the trouble areas, a copy of the config you used and a 3d model you see the problem with using the supplied configuration.

**Comment by mrvn** [comment]
And what version of slic3r. With or without small wall detection?

**Comment by lordofhyphens** [comment]
https://github.com/slic3r/Slic3r/blob/master/utils/post-processing/decimate.pl also exists.
